.gitignore

```diff
@@ -1,3 +1,11 @@
 .idea
 **/target
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+server/bin/.project
+server/bin/.settings/org.eclipse.core.resources.prefs
+server/bin/.settings/org.eclipse.m2e.core.prefs
+server/bin/pom.xml
+server/bin/server.yaml
+server/bin/skaffold.yaml
+server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class
+server/bin/src/main/java/com/example/server/PingServer.class
```

Comment:
Please find a way to ignore all of these files using a single matching statement

Reply:
Done with the latest commit.